cylc/uiserver/config_defaults.py

```diff
@@ -19,8 +18,10 @@
 from pathlib import Path
 import pkg_resources
 
-HERE = Path(__file__).resolve().parent
-DIST = HERE.joinpath(Path('../cylc-ui/dist'))
+from cylc.uiserver import __file__ as uis_pkg
+
+
+DIST = Path(uis_pkg).parents[3] / 'cylc-ui/dist'
```

Comment:
Had to setup a link here:
```
(uiserver) sutherlander@cortex-vbox:python3.7$ pwd
/home/sutherlander/.envs/uiserver/lib/python3.7
(uiserver) sutherlander@cortex-vbox:python3.7$ ls -l
total 12
lrwxrwxrwx   1 sutherlander sutherlander    31 Feb 24 01:09 cylc-ui -> /home/sutherlander/cylc/cylc-ui
drwxr-xr-x 142 sutherlander sutherlander 12288 Feb 24 00:56 site-packages
```

I suppose there's a better way of doing this..

Reply:
Ah, ok, I'm guessing you've not installed the UIS in editable mode so `cylc.uiserver.__file__` is showing up as being within the python lib rather than the UIS working copy.

I'm not sure how we are going to install the UI into the UIS yet, I think `DIST` should be configurable somehow.

Reply:
You should now be able to set the static path for your UI installation using the user config like so:

```python
#  ~/.cylc/hub/config.py
c.Spawner.cmd = ['cylc', 'uiserver', '-s', '<path>']
```